crates/shared/src/bad_token/trace_call.rs

```diff
@@ -468,102 +403,151 @@ mod tests {
-            H160(hex!("389999216860AB8E0175387A0c90E5c52522C945")),
-            H160(hex!("45734927Fa2f616FbE19E65f42A0ef3d37d1c80A")),
-            H160(hex!("4F9254C83EB525f9FCf346490bbb3ed28a81C667")),
-            H160(hex!("51d3e4C0b2c83E62f5d517D250b3e856897d2052")),
-            H160(hex!("925f2C11b99c1A4c46606898ee91eD3D450cFedA")),
-            H160(hex!("97b65710D03E12775189F0D113202cc1443b0aa2")),
-            H160(hex!("f198B4a2631B7D0B9FAc36f8B546Ed3DCe472A47")),
-            H160(hex!("1016f3c0a1939fa27538339da7e2a300031b6f37")),
-            H160(hex!("7f0f118d083d5175ab9d2d34c4c8fa4f43c3f47b")),
-            H160(hex!("d3f6571be1d91ac68b40daaa24075ca7e2f0f72e")),
-            H160(hex!("bf25ea982b4f850dafb4a95367b890eee5a9e8f2")),
-            H160(hex!("c73c167e7a4ba109e4052f70d5466d0c312a344d")),
-            H160(hex!("1426cc6d52d1b14e2b3b1cb04d57ea42b39c4c7c")),
-            H160(hex!("99ddddd8dfe33905338a073047cfad72e6833c06")),
-            H160(hex!("4a6be56a211a4c4e0dd4474d524138933c17f3e3")),
-            H160(hex!("9f41da75ab2b8c6f0dcef7173c4bf66bd4f6b36a")),
-            H160(hex!("239dc02a28a0774738463e06245544a72745d5c5")),
-            H160(hex!("801ea8c463a776e85344c565e355137b5c3324cd")),
-            H160(hex!("aee53701e18d5ff6af4964c3a381e7d09b9b9075")),
-            H160(hex!("3a6fe4c752eb8d571a660a776be4003d619c30a3")),
-            H160(hex!("99043bb680ab9262c7b2ac524e00b215efb7db9b")),
-            H160(hex!("18bdfc80b97cb97f6b466cce967849ce9cd9d58c")),
-            H160(hex!("32c868f6318d6334b2250f323d914bc2239e4eee")),
-            H160(hex!("c626d951eff8e421448074bd2ad7805c6d585793")),
-            H160(hex!("94987bc8aa5f36cb2461c190134929a29c3df726")),
-            H160(hex!("41933422dc4a1cb8c822e06f12f7b52fa5e7e094")),
-            H160(hex!("b893a8049f250b57efa8c62d51527a22404d7c9a")),
-            H160(hex!("f198b4a2631b7d0b9fac36f8b546ed3dce472a47")),
-            H160(hex!("76851a93977bea9264c32255b6457882035c7501")),
-            H160(hex!("a5959e9412d27041194c3c3bcbe855face2864f7")),
-            H160(hex!("cf8c23cf17bb5815d5705a15486fa83805415625")),
-            H160(hex!("63d0eea1d7c0d1e89d7e665708d7e8997c0a9ed6")),
-            H160(hex!("98ecf3d8e21adaafe16c00cc3ff681e72690278b")),
-            H160(hex!("3ea50b7ef6a7eaf7e966e2cb72b519c16557497c")),
-            H160(hex!("2b1fe2cea92436e8c34b7c215af66aaa2932a8b2")),
-            H160(hex!("c7c24fe893c21e8a4ef46eaf31badcab9f362841")),
-            H160(hex!("ef5b32486ed432b804a51d129f4d2fbdf18057ec")),
-            H160(hex!("2129ff6000b95a973236020bcd2b2006b0d8e019")),
-            // Should be denied because can't approve more than balance
-            H160(hex!("decade1c6bf2cd9fb89afad73e4a519c867adcf5")),
-            // All balances are maxuint256
-            H160(hex!("0027449Bf0887ca3E431D263FFDeFb244D95b555")),
+            addr!("0027449Bf0887ca3E431D263FFDeFb244D95b555"), // All balances are maxuint256
```

Comment:
Sorted list, added some new known bad tokens, and replaced `H160(hex!(...))` with `addr!(...)`.

Previously tested tokens should all still be there.